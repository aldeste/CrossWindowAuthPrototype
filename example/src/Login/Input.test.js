import React from "react";
import Input from "./Input";
import ReactShallowRenderer from "react-test-renderer/shallow";
const renderer = new ReactShallowRenderer();

describe("Input react component", () => {
  const tree = renderer.render(<Input />);
  it("Is a react component", () => {
    expect(typeof Input).toBe("function");
    expect(tree.$$typeof.toString()).toBe("Symbol(react.element)");
  });

  it("Matches earlier configuration", () => {
    expect(tree).toMatchSnapshot();
  });
});
