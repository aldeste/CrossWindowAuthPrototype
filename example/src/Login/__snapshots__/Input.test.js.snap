// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input react component Matches earlier configuration 1`] = `
<styled.div>
  <styled.label
    htmlFor={undefined}
    required={false}
  />
  <styled.input />
</styled.div>
`;
